join-statements/joinstatements.md

```diff
@@ -0,0 +1,423 @@
+1) To find data held in two separate data tables, we could do separate searches on each table, _or_ we could use a `JOIN` statement in order to call different data from different tables into a query.
+
+2) Assuming 2 tables, A and B:
+
+**CROSS JOIN** - it shows all possible combinations between rows from table A and rows from table B, so a result from the query would be <# of A rows> x <# of B rows>
+
+A real world example would be if I had a table with fruit juices, and another table with entrees at a high school cafeteria. A CROSS JOIN would allow me to compare all the possible combinations of meals for the high schoolers.
+
+```
+SELECT m.dish, d.drink
+FROM meals AS m
+CROSS JOIN drink AS d;
+```
+
+**INNER JOIN** - it's the result of a query between 2 tables where the output is the intersection of table A and table B that satisfies a specified conditional statement between both tables using the `ON` statement.
+
+A real world example would be if, there were a table of students being specified in the `FROM` portion of the query, and another table of different extracurricular societies that keeps records of students by their student ID. We could use the `INNER JOIN`, to get a list of students that are found in a given society using an INNER JOIN between these two tables. The `INNER JOIN` would return matches as specified in the `ON` statement. The query would look something like this.
+
+```
+SELECT students.name societies.society
```

Comment:
your query doesn't match what you say your output is

Reply:
done